matchbox-server/src/main/java/ch/ahdis/matchbox/packages/IgLoaderFromJpaPackageCache.java

```diff
@@ -176,26 +176,26 @@ public void loadIg(List<ImplementationGuide> igs, Map<String, ByteProvider> bina
 
 		switch (FhirVersionEnum.forVersionString(this.getVersion())) {
 			case R4, R4B -> {
-				if (src.startsWith("hl7.terminology.") && !PACKAGE_R4_TERMINOLOGY.equals(src)) {
-					log.debug("Replacing '{}' with '{}'", src, PACKAGE_R4_TERMINOLOGY);
+				if (src.startsWith("hl7.terminology#6.3.0") || PACKAGE_R4_TERMINOLOGY.equals(src)) {
```

Comment:
That means Matchbox will stop replacing previous versions (e.g. 6.2.0) with the bundled one, leading to multiple versions being loaded. Is that what we want?

Reply:
yes, that's the feedback from the WGM, we need to support multiple terminology versions, and I would suggest we include in matchbox just the latest but don't replace the latest